x-pack/plugins/lens/public/drag_drop/providers/providers.tsx

```diff
@@ -220,16 +127,17 @@ export function RootDragDropProvider({ children }: { children: React.ReactNode }
 }
 
 export function nextValidDropTarget(
-  activeDropTarget: DropTargets | undefined,
+  dropTargetsByOrder: RegisteredDropTargets,
+  activeDropTarget: DropIdentifier | undefined,
   draggingOrder: [string],
   filterElements: (el: DragDropIdentifier) => boolean = () => true,
   reverse = false
 ) {
-  if (!activeDropTarget) {
+  if (!dropTargetsByOrder) {
     return;
   }
 
-  const filteredTargets = [...Object.entries(activeDropTarget.dropTargetsByOrder)].filter(
+  const filteredTargets = [...Object.entries(dropTargetsByOrder)].filter(
```

Comment:
I think you can avoid to spread here and obtain the same result (but saving some memory + cpu cycles).